### PR TITLE
Enrich HTML5-History-API's package.json(field:license)

### PR DESCRIPTION
**Pull request by iwdmb**
#5500

license link:
1. [https://github.com/devote/HTML5-History-API/blob/master/package.json](https://github.com/devote/HTML5-History-API/blob/master/package.json)
2. [https://github.com/devote/HTML5-History-API/blob/master/LICENSE-MIT](https://github.com/devote/HTML5-History-API/blob/master/LICENSE-MIT)
3. [https://github.com/devote/HTML5-History-API/commit/825d10706749e918a15c388e435365340743bf93](https://github.com/devote/HTML5-History-API/commit/825d10706749e918a15c388e435365340743bf93)

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@iwdmb thanks. @pvnr0082t please help review this PR.

File: ajax/libs/html5-history-api/package.json [hunk 1]
```diff
@@ -30,7 +30,6 @@
     "url": "git://github.com/devote/HTML5-History-API.git"
   },
   "licenses": [
-    "GPL-3.0",
     "MIT"
   ],
```
Comment (pvnr0082t) [inline ajax/libs/html5-history-api/package.json]: Let's use `"license": "MIT",` here. Thanks.

**Comment by pvnr0082t** [comment]
Any update? @iwdmb

**Comment by PeterDaveHello** [comment]
Moved to #9808

**Pull request closed without merging**